linked_list/ll_insertions.py

```diff
@@ -0,0 +1,26 @@
+from node import Node
+from linked_list import LinkedList 
+
+
+class ll_methods(LinkedList):
```

Comment:
It is alright to just add these methods to the `LinkedList` class directly if you wish.